pkgs/top-level/all-packages.nix

```diff
@@ -16533,8 +16533,6 @@ with pkgs;
 
   cadre = callPackage ../development/tools/cadre { };
 
-  cask = callPackage ../development/tools/cask { };
```

Comment:
Why remove the top level attribute? This would at least need a changelog entry, but does it even make sense?

Reply:
IMHO it makes no sense to promote Cask to top-level, because it is utterly tied to Emacs. Indeed, the old expression explicitly called the internal structure of our code generation.

But I can put this in `aliases.nix`.